apps/aecore/src/aec_block_generator.erl

```diff
@@ -91,13 +107,13 @@ handle_cast(start_generation, State) ->
     {noreply, do_start_generation(State)};
 handle_cast({worker_done, Pid, {candidate, Candidate, CandidateState}},
             State = #state{ worker = {Pid, _} }) ->
-    %% Only publish non-empty micro-blocks
+    %% Only publish non-empty microblocks
     case aec_blocks:txs(Candidate) of
         [] ->
-            lager:debug("New empty microblock candidate generated, and discarded", []),
+            lager:debug("Empty microblock candidate prepared", []),
             ok;
         _  ->
-            epoch_mining:info("New microblock candidate generated", []),
+            epoch_mining:info("New microblock candidate ready", []),
```

Comment:
Not an improvement?

Reply:
Well, it's getting published as being ready to fetch. Tried to make it contrast with the alternative message above